src/cpp/src/image_generation/flux_pipeline.hpp

```diff
@@ -477,6 +500,7 @@ class FluxPipeline : public DiffusionPipeline {
     std::shared_ptr<T5EncoderModel> m_t5_text_encoder = nullptr;
     std::shared_ptr<AutoencoderKL> m_vae = nullptr;
     ImageGenerationConfig m_custom_generation_config;
+    ImageGenerationPerfMetrics m_perf_metrics;
```

Comment:
should it be moved to base `DiffusionPipeline` class? All derived pipelines will inherit this field.